### ISSUE TITLE
Synchronization of shared mailboxes on Office 365 is not possible

### ISSUE THREAD

**Issue by romainlombart** [jobisoft/EAS-4-TbSync#48]
Windows 10 Pro 64 bits
TbSync version: 0.7.22
Thunderbird version: 60.4.0 (32bits)

Hello !

For shared mailboxes, the user must be for an example :
johndoe@domain.com\sales@domain.com
[https://computerz.solutions/office-365-boite-partagee-smartphone/](url)

The error is "Autodiscover needs a valid email address as user name"

Except that in this case this address is correct on Office 365, it must be configured like this in order to retrieve imap emails for example.
Can you add this type of address so that it is possible to set up these shared Office 365 accounts?

**Comment by jobisoft** [comment]
Moved this to the EAS provider. I will have a look.

**Comment by jobisoft** [comment]
I am the admin of an O365 instance and I checked this and it does not work. It works for IMAP, but not for EAS. Not only does the autodiscover not work, but also the manual setup does not work. EAS does not accept such a username (after the \  you usually place the exchange alias and not the full email, but I tried both).

O365 allows to actually share calendars and address books with other members of the same instance and that is propagated to EAS.
